rpcs3/rpcs3qt/mainwindow.cpp

```diff
@@ -0,0 +1,233 @@
+#ifdef QT_UI
```

Comment:
Yeah, wrong line. But there is a handful of include guard in other files, like `#ifndef CORETAB_H`.
Anyway, this whole qt part seems to follow Qt style (in naming and such), so I guess it's fine for now.